src/ctap2/commands/get_assertion.rs

```diff
@@ -1155,7 +1146,7 @@ pub mod test {
 
         // Sending first GetAssertion with first allow_list-entry, that will return an error
         let mut msg = cid.to_vec();
-        msg.extend(vec![HIDCmd::Cbor.into(), 0x00, 0x94]);
+        msg.extend(vec![HIDCmd::Cbor.into(), 0x00, 0x90]);
```

Comment:
Not a huge fan of "this is the magic length byte (I assume?) that you have to know to modify if you change anything below", but I think that's out of scope for this change.

Reply:
Agreed, I filed https://github.com/mozilla/authenticator-rs/issues/298.